src/styles/main.scss

```diff
@@ -1,3 +1,80 @@
+$month-length: 28, 29, 30, 31;
+$start-day: (
+  "mon": 0,
+  "tue": 1,
+  "wed": 2,
+  "thu": 3,
+  "fri": 4,
+  "sat": 5,
+  "sun": 6
+);
+
+$main-indent: 10px;
+$gap: 1;
+$base-size: 100px;
+$columns: 7;
+
+@function size($block-size, $indent, $gap, $col) {
+  $sum: $col * ($block-size + $gap ) + ($indent * 2);
+
+  @return $sum;
+}
+
+@function firstDayIndent($week, $size, $gap) {
+  $result: $week * ($size + $gap);
+
+  @return $result;
+}
+
 body {
   margin: 0;
 }
+
+.calendar {
```

Comment:
Add to this class `margin: 0 auto;` and your calendar is aligned horizontally in the center